data/campaigns/Sceptre_of_Fire/scenarios/9_Caverns_of_Flame.cfg

```diff
@@ -867,11 +1078,25 @@
                 fire_event=no
             [/kill]
             {FADE_TO_BLACK}
-            [message]
-                speaker=narrator
-                message= _ "Once the magic seal capping the throat of the volcano was removed, lava and toxic gas exploded upward, killing everyone inside and sealing them in a tomb of hot, flowing rock. The Sceptre of Fire would wait, unharmed by the lava, until some day far in the future."
-                image=wesnoth-icon.png
-            [/message]
+            [if]
+                [have_unit]
+                    type=Gryphon Rider
```

Comment:
Should be
```ini
type=Gryphon Rider,Gryphon Master
```

Reply:
Oh, I missed that - and I tested with a Gryphon Master...